src/customFields.gs

```diff
@@ -1,17 +1,14 @@
+// Node required code block
+const debug = require("./debug.gs").debug;
+const EpicField = require("./models/jira/EpicField.gs");
+const UserStorage = require("./UserStorage.gs").UserStorage;
+const getAllJiraFields = require("./jiraCommon.gs").getAllJiraFields;
+// End of Node required code block
+
 var CUSTOMFIELD_FORMAT_RAW    = 1;
 var CUSTOMFIELD_FORMAT_SEARCH = 2;
 var CUSTOMFIELD_FORMAT_UNIFY  = 3;
 
-// storage of custom Jira field 'EPIC'
-var fieldEpic = {
-  usable:    false,  // true|false
-  key:       'jst_epic',
-  name:      'Epic',
-  link_key:  null, // customfield_10003
-  label_key: null  // customfield_10005
-};
-
-
```

Comment:
The management of storing the EpicField has been moved into a single model class

Reply:
As I work more on other issues - I think a model that looks after all Field Definitions may be a good idea. But thats another piece of work as we move the code base to the structure we discussed in the other Github issue